ses/client.bal

```diff
@@ -0,0 +1,519 @@
+        map<string> signedRequestHeaders = check getSignedRequestHeaders(self.host, self.accessKey, self.secretKey,
+                                                                         self.region, GET, requestUri, EMPTY_STRING);
+        http:Response httpResponse = check self.awsSesClient-> get(requestUri, signedRequestHeaders);
+        return (check handleHttpResponse(httpResponse)).cloneWithType(ContactList);
+    }
+
+    # Lists all of the contact lists available.
+    # + return - An aws.ses:ContactLists record on success else an error 
+    @display {label: "List Contact Lists"}
+    remote isolated function listContactLists() returns @display {label: "All Contact Lists"} ContactLists|error{
+        string requestUri = URI + "contact-lists";
+        map<string> signedRequestHeaders = check getSignedRequestHeaders(self.host, self.accessKey, self.secretKey,
+                                                                         self.region, GET,requestUri,EMPTY_STRING);
+        http:Response httpResponse = check self.awsSesClient->get(requestUri, signedRequestHeaders);
+        return (check handleHttpResponse(httpResponse)).cloneWithType(ContactLists);
+    }
+
+    # Deletes a contact list and all of the contacts on that list.
+    #
+    # + contactListName - The name of the contact list
+    # + return - An empty json on success else an error
+    @display {label: "Delete Contact List"}
+    remote isolated function deleteContactList(@display {label: "Contact List Name"} string contactListName)
+                                               returns @display {label: "Contact List Delete Response"} json|error {
+        string requestUri = URI + "contact-lists/"+contactListName;
+        map<string> signedRequestHeaders = check getSignedRequestHeaders(self.host, self.accessKey, self.secretKey,
+                                                                         self.region, DELETE, requestUri, EMPTY_STRING);
+        http:Response httpResponse = check self.awsSesClient-> delete(requestUri, headers= signedRequestHeaders);
+        return check handleHttpResponse(httpResponse);      
+    }
+
+    # Creates a contact, which is an end-user who is receiving the email, and adds them to a contact list.
+    #
+    # + contactListName - The name of the contact list
+    # + contactCreationRequest - The request payload to create contact
+    # + return - An empty json on success else an error
+    @display {label: "Create Contact"}
+    remote isolated function createContact(@display {label: "Contact List Name"} string contactListName,
+                                           @display {label: "Request To Create Contact"}
+                                           ContactCreationRequest contactCreationRequest) 
+                                           returns @display {label: "Contact Creation Response"} json|error {
+        string requestUri = URI + "contact-lists/" + contactListName + "/contacts";
```

Comment:
Shall we create a const variable for `"contact-lists/"` and `"/contacts"` and reuse them? It is used in many places